default/stringtables/en.txt

```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
 
 [[GROWTH_SPECIALS_ENTRY_LIST]]'''
 
+INDUSTRY_FOCUS_TITLE
+Industry Focus
+
+INDUSTRY_FOCUS_TEXT
+The Industry Focus represents a plantey primarily focused on industrial production. [[metertype METER_INDUSTRY]] will receive a bonus which increases as the planetary population increases. This is the default focus for any species that does not have a preferred focus.
+
 INFLUENCE_FOCUS_TITLE
 Influence Focus
 
 INFLUENCE_FOCUS_TEXT
 Influence isn't implemented yet.
 
+LOGISTICS_FOCUS_TITLE
+Logistics Focus
+
+LOGISTICS_FOCUS_TEXT
+By focusing on Logistics, a planet is able to vastly extend its supply. The [[metertype METER_SUPPLY]] will be increased by a factor of three.
+
+PLANETDRIVE_FOCUS_TITLE
+Planet Drive Focus
+
+PLANETDRIVE_FOCUS_TEXT
+After a [[BLD_PLANET_DRIVE]] has been constructed, and a [[buildingtype BLD_PLANET_BEACON]] or [[shippart SP_PLANET_BEACON]] is in an adjacent system, selecting the [[PLANETDRIVE_FOCUS_TITLE]] allows the planet to move between star systems.
+
 PROTECTION_FOCUS_TITLE
 Protection Focus
 
 PROTECTION_FOCUS_TEXT
 The Protection focus represents a planet wholly focused on defending itself. [[metertype METER_TROOPS]], Planetary [[metertype METER_DEFENSE]], and Planetary [[metertype METER_SHIELD]] are doubled.
 
+PSYCHIC_FOCUS_TITLE
+Psychic Domination Focus
+
+PSYCHIC_FOCUS_TEXT
+'''The [[PSYCHIC_FOCUS_TITLE]] concentrates the mental force exerted by the entire population to dominate the thoughts of non-telepathic crews of ships in the same system.
+
+Each turn there is a 10% chance of taking control of of their ships unless the enemy empire also posesses the technology [[LRN_PSY_DOM]].
+
+When the Psychic Domination Focus is engaged it also reduces the chance that [[predefinedshipdesign SM_PSIONIC_SNOWFLAKE]]s take over your ships.'''
```

Comment:
...engaged, it also...

...[[predefinedshipdesign SM_PSIONIC_SNOWFLAKE]]s will take control of ships controlled by the planet's empire.